app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java

```diff
@@ -305,5 +323,43 @@ private boolean noDuplicateNumber() {
         }
         return noDuplicate;
     }
+
+    /**
+     * Set focus to the next EditText after picking contact
+     */
+    public void setFocusOnNextView() {
```

Comment:
For finding the index of the selected button try using tag of EditText (phoneInput) instead of introducing new tags for the buttons.